Error: The `form` element was not requested. Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group/4/9d34326d-443a-46a2-9521-644f0741030c.html b/output/Checkbox Group/4/9d34326d-443a-46a2-9521-644f0741030c.html
index f5f3493..06dfeec 100644
--- a/output/Checkbox Group/4/9d34326d-443a-46a2-9521-644f0741030c.html	
+++ b/output/Checkbox Group/4/9d34326d-443a-46a2-9521-644f0741030c.html	
@@ -1,17 +1,15 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <div>
-      <input type="checkbox" id="bananas" name="fruits" value="bananas">
-      <label for="bananas">Bananas</label>
-    </div>
-    <div>
-      <input type="checkbox" id="oranges" name="fruits" value="oranges">
-      <label for="oranges">Oranges</label>
-    </div>
-    <div>
-      <input type="checkbox" id="apples" name="fruits" value="apples">
-      <label for="apples">Apples</label>
-    </div>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>What fruits do you like?</legend>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
